### PR TITLE
 Improve links to Learn Chef Rally

### PR DESCRIPTION
**Pull request by tpetchel**
We recently published some updates to Learn Chef Rally that we'd like to highlight.

### CODE REVIEW

**Review by chris-rock — APPROVED** [review]
Thank you @tpetchel for that improvement

File: www/source/tutorials.html.slim [hunk 1]
```diff
@@ -20,8 +20,21 @@ header.child-h.blue-gradient.margin-top-offset
...
+          i.focus &GreaterGreater;
+          |  Compliance Automation with InSpec
+          div  Start here. Downloadable training environments get you started with everything you need.
+      li
+        a.tutorials--link href="https://learn.chef.io/tracks/integrated-compliance/"
+          i.focus &GreaterGreater;
+          |  Integrated Compliance with Chef
+          div  Continue your journey here if you're interested in using Chef to correct potential compliance and security issues.
     br
     p.clear Huge thanks to Annie Hedgpeth for creating all of these fantastic tutorials.
```
Comment (adamleff) [inline www/source/tutorials.html.slim:39]: To make it clear which of the tutorials are Annie's, I suggest changing this line to:

"Huge thanks to Annie Hedgpeth for creating all of the following fantastic tutorials:"

**Review by adamleff — CHANGES_REQUESTED** [review]
Changes overall look good. One wording modification I'd like to see for clarity before merge/release.

**Comment by chris-rock** [comment]
@tpetchel Anything we can help to get this in?

**Comment by adamleff** [comment]
This is a good change as-is so I'll merge it and then put forth a quick one-liner change to update.

**Pull request merged**